SU-SWS/stanford_news
# D8CORE-3515: fixing news padding for no image

# READY FOR REVIEW

# Summary
- Moved where the padding is applied so that the link doesn't get padded twice.
- Also need to verify that the image does not get any new padding

# Needed By (Date)
- 2/26

# Urgency
- Medium

# Steps to Test

1. Pull in this change.
2. clear caches.
3. Create news articles that have an external and internal link without an image.
4. Create a news article that has an image for comparison. 
5. View the articles as cards in the following locations.
     1. List of News as a list- verify the padding looks right
     2. List of News as a card list 
     3. News listing page /news
     4. A news article with the cards in the Read more news.

# Affected Projects or Products
- News

# Associated Issues and/or People
- D8CORE-3515

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
